Respawn/SqlServerDbAdapter.cs

```diff
@@ -185,7 +185,7 @@ public string BuildTurnOnSystemVersioningCommandText(IEnumerable<TemporalTable>
             var builder = new StringBuilder();
             foreach (var table in tablesToTurnOnSystemVersioning)
             {
-                builder.Append($"alter table {table.Schema}.{table.Name} set (SYSTEM_VERSIONING = ON (HISTORY_TABLE = {table.HistoryTableSchema}.{table.HistoryTableName}));\r\n");
+                builder.Append($"alter table \"{table.Schema}\".\"{table.Name}\" set (SYSTEM_VERSIONING = ON (HISTORY_TABLE = {table.HistoryTableSchema}.{table.HistoryTableName}));\r\n");
```

Comment:
You can use the `QuoteCharacter` constant here to make this a bit more consistent.

Reply:
Fixed.